pkg/globalmanager/controllers/jointinference/jointinferenceservice.go

```diff
@@ -62,7 +62,7 @@ const (
 )
 
 // Kind contains the schema.GroupVersionKind for this controller type.
-var Kind = sednav1.SchemeGroupVersion.WithKind(Name)
+var Kind = sednav1.SchemeGroupVersion.WithKind(KindName)
 
```

Comment:
The bug fix and documentation PRs should be submitted separately. Please keep this in mind for future submissions.

Reply:
Yes, I get it.Thank you, Mr. Tang.